packages/editor/src/browser/editor-preferences.ts

```diff
@@ -32,12 +32,14 @@ export const editorPreferenceSchema: PreferenceSchema = {
             'type': 'number',
             'minimum': 1,
             'default': 4,
-            'description': 'Configure the tab size in the editor.'
+            'description': 'Configure the tab size in the editor.',
+            'scope': 'resource'
```

Comment:
VS Code allows to define default scope for schema: https://github.com/Microsoft/vscode/blob/6b8a54df78848c1fa582557fc4ca583d67177a56/src/vs/editor/common/config/commonEditorConfig.ts#L227

Could we do the same instead of copy & paste?

Reply:
Yes. Done in the latest verison.